### ISSUE TITLE
Text writer is not adding single quotes when writing annotations with booleans

### ISSUE THREAD

**Issue by raganhan** [amzn/ion-js#206]
```
writer.writeInt(1, ["false"]);
writer.close();
console.log(ion.decodeUtf8(writer.getBytes()));
```

outputs `false::1` when it should be `'false'::1` as `false` is an Ion boolean an annotations **must** be symbols

**Comment by raganhan** [comment]
This works fine for numbers, but it's best to add tests for all data types 

```
writer.writeInt(1, ["1"]);
writer.close();
console.log(ion.decodeUtf8(writer.getBytes()));
```
outputs `'1'::1`

**Comment by pbcornell** [comment]
Addressed in #263, closing.

```javascript
writer.writeInt(1, ["false"]);
writer.close();
console.log(ion.decodeUtf8(writer.getBytes()));
```
now correctly produces:
```
'false'::1
```
